src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -462,8 +396,97 @@ paths:
               application/xml:
                 schema:
                   $ref: '#/components/schemas/IIIF_info'
+
+  
+  # DeepZoom
+  /deepzoom/{imageid}.dzi:
+    get:
+      tags:
+        - 'Access'
+      summary: 'DeepZoom Image'
+      description: |- 
+        Deep Zoom provides the ability to interactively view high-resolution images. You can zoom in and out of images rapidly without affecting the performance of your application. Deep Zoom enables smooth loading and panning by serving up multi-resolution images and using spring animations.
+      operationId: getDeepzoomImage
+      parameters:
+      - name: imageid
+        in: path
+        description: "Identifier/path for image."
+        required: true
+        schema:
+          type: string
+        example: "image/path.tif"
+      
+      responses:
+        '200':
+            description: 'Succes!'
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/DeepzoomDZI'
+              application/xml:
+                schema:
+                  $ref: '#/components/schemas/DeepzoomDZI'
+                  
+  /deepzoom/{imageid}_files/{layer}/{tiles}.{format}:
+    get:
+      tags:
+        - 'Access'
+      summary: 'DeepZoom Tile'
+      description: |- 
+        DeepZoom can bee used with the Internet Imaging Protocol (IIP). This endpoint only requires the DeepZoom parameter to work. Besides, this endpoint has the capability to make use of the IIP parameters shown below. 
+      operationId: getDeepzoomTile
+      parameters:
+      - name: imageid
+        in: path
+        description: 'Identifier/path for image.'
+        required: true
+        schema:
+          type: string
+        example: "image/path.tif"
+      - name: layer
```

Comment:
Could you set the a default here? I suggest `8` since 2^8 = 256, meaning a tile shows the whole (downscaled) image.

Reply:
Default has been set to 8